tools/makepy_pinmapinterfaces.py

```diff
@@ -1,11 +1,22 @@
 import sys
 import os.path
+import winreg
 from win32com.client import makepy
 
+
```

Comment:
We use this file to generate `_pinmapinterfaces.py`. I only expect we will run it once during every new major version release of TSM.  I also updated this file in https://github.com/ni/nitsm-python/pull/158/files#diff-bef4b1f2d637a94964b183d0743dd4b276b814f6c39b4c15e43927589ecf8373.  My update doesn't require querying the registry, but does require changing the version number when it is run.  Thoughts on which is better of the two approaches?  I'm inclined to let pywin32 discover the type libraries for us.

Reply:
Ah, ok. That makes sense. Unlike the change in `systemtests/conftest.py`, this one wasn't necessary in order to get the tests to pass. I guess I wasn't sure exactly when this file ran, but either way I thought it was bad to hardcode the TSM version number here, so I made the change.

Based on what you're saying, however, there isn't really a need to get the environment variable from the registry. Since this code doesn't run in the pipeline, it should be sufficient to use os.environ.

So my preference would still be to not require a manual change each time we want to update the version, but to use os.environ instead of getting it from the registry. What do you think? I'm also ambivalent about doing it in this PR or [in yours](https://github.com/ni/nitsm-python/pull/158).

Reply:
Perfect, this is definitely the option since it only requires we use version selector prior to generating `_pinmapinterfaces.py`.